examples/docker-client/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 Flower Labs GmbH. All Rights Reserved.
+FROM flwr/client:1.6.0-py3.8-ubuntu22.04
+
+WORKDIR /app
+COPY requirements.txt .
+RUN python -m pip install -U --no-cache-dir -r requirements.txt
+
+COPY client.py .
+ENTRYPOINT ["python", "-c", "from flwr.client import run_client; run_client()", "--callable", "client:flower"]
```

Comment:
We can define a `poetry.script` in the pyproject file as we do in the flower `pyproject` file. 
wdyt? 

However, it will require us to install poetry in the dockerfile.

Reply:
Shouldn't flower-client exist when installing `flwr` as it's already present here: https://github.com/adap/flower/blob/main/pyproject.toml#L58

Reply:
I'm not sure if this is the case when we install `flwr` via pip. I will check.

Reply:
if you access the container and reisntall `flwr` it will work. i think the problem is lack of user in the container?